.eslintrc

```diff
@@ -167,7 +167,7 @@
     "func-style": 0, //enforce use of function declarations or expressions
     "id-length": 0, //this option enforces minimum and maximum identifier lengths (variable names, property names etc.)
     "id-match": [0, "^[a-z]+([A-Z][a-z]*)*$", { "properties": true }], //require identifiers to match the provided regular expression
-    "indent": [1, 2], //specify tab or space width for your code (fixable)
+    // "indent": [1, 2], //specify tab or space width for your code (fixable)
```

Comment:
If I well remember, it didn't accept the indentation of my switch-case in reducers folder.
But they were auto-formatted by prettier, so totally legit I guess.

In fact, if your prettier is well configured (which is normally thanks to the prettierrc file at the root) you don't need your linter to check your indentations.

Reply:
That's what I thought. Actually I asked because I had same problem in another language 2 days ago, and had to configure for switch with the option {"SwitchCase": 1}. But I'm good with using this well done prettier config!